invenio_vocabularies/contrib/affiliations/config.py

```diff
@@ -26,7 +26,16 @@ class AffiliationsSearchOptions(SearchOptions):
     """Search options."""
 
     suggest_parser_cls = SuggestQueryParser.factory(
-        fields=["name^100", "acronym^50", "title.*^20", "id^20", "aliases^20"],
+        fields=[
+            "name^100",
+            "acronym.keyword^100",
+            "acronym^40",
+            "title.*^20",
```

Comment:
This might be an issue for search inputs that are more than 5 words, since it will give a `too_many_nested_clauses` error. The reason is that there are ~200 languages in the mapping, so if you search for `"one two three four five"`, this will generate 5 * 200 search clauses.

An alternative to this is what @zzacharo suggested, which is to dynamically use the user's locale when searching, i.e. `title.{get_locale()}`.

Reply:
Fixed by https://github.com/inveniosoftware/invenio-records-resources/pull/594

Reply:
There are some comments on https://github.com/inveniosoftware/invenio-records-resources/pull/594 . We, with @0einstein0 , will put this in progress and you can move it back when the dependent PR comments are addressed

Reply:
I have closed the PR in `invenio-records-resource`.